DigitalLearningSolutions.Web/ViewModels/Register/SummaryViewModel.cs

```diff
@@ -2,33 +2,27 @@
 {
     using System.Collections.Generic;
     using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Web.ViewModels.MyAccount;
 
-    public class SummaryViewModel: IValidatableObject
+    public class SummaryViewModel : IValidatableObject
     {
-        public SummaryViewModel() { }
-
-        public SummaryViewModel(string firstName, string lastName, string email, string centre, string jobGroup)
-        {
-            FirstName = firstName;
-            LastName = lastName;
-            Email = email;
-            Centre = centre;
-            JobGroup = jobGroup;
-        }
         public string? FirstName { get; set; }
         public string? LastName { get; set; }
         public string? Email { get; set; }
         public string? Centre { get; set; }
         public string? JobGroup { get; set; }
         public bool Terms { get; set; }
+        public IEnumerable<EditCustomFieldViewModel>? CustomFields { get; set; }

```

Comment:
Ah - do we want to pull the EditCustomFieldVM from MyAccount to somewhere shared/do we need all the fields you need for editing (isn't there a whole list of options)?